src/pages/Landing/components/EventsCalendarView.tsx

```diff
@@ -15,6 +12,8 @@ import { useAnalytics } from 'hooks/Utils';
 
 import Paper from 'components/layout/Paper';
 
+import EventsCalendarPopover from './EventsCalendarPopover';
```

Comment:
Bruk absolutt import
```suggestion
import EventsCalendarPopover from 'pages/Landing/components/EventsCalendarPopover';
```